src/main/java/uk/nhs/digital/nhsconnect/nhais/mesh/MongoScheduler.java

```diff
@@ -21,16 +21,16 @@
     private final SchedulerTimestampRepositoryExtensions schedulerTimestampRepository;
 
     @Value("${nhais.scheduler.intervalInSeconds}")
-    private long seconds;
+    private long seconds = 10;
```

Comment:
Why `10` here?

Reply:
because I've forgotten to remove change made to run it  locally